libs/ngx-charts-on-fhir/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elimuinformatics/ngx-charts-on-fhir",
-  "version": "7.0.1",
+  "version": "7.0.3",
```

Comment:
this should be a major version change because required angular version is now 19+